GeosCore/carbon_gases_mod.F90

```diff
@@ -1241,21 +1035,16 @@ SUBROUTINE Init_Carbon_Gases( Input_Opt,  State_Chm, State_Diag,             &
 
     !========================================================================
     ! Define GEOS-Chem species indices
-    ! NOTE: Some of these are for tagged species, which are deactivated
-    ! by default.  Interested users can add tagged species if they wish.
     !========================================================================
-    id_CH4     = Ind_( 'CH4'     )
-    id_CO      = Ind_( 'CO'      )
-    id_COacet  = Ind_( 'COacet'  )
-    id_COch3oh = Ind_( 'COch3oh' )
-    id_COch4   = Ind_( 'COch4'   )
-    id_COisop  = Ind_( 'COisop'  )
-    id_COmono  = Ind_( 'COmono'  )
-    id_COnmvoc = Ind_( 'COnmvoc' )
-    id_CO2     = Ind_( 'CO2'     )
-    id_CO2ch   = Ind_( 'CO2ch'   )
-    id_OCS     = Ind_( 'OCS'     )
-    id_OH      = Ind_( 'FixedOH' )
+    id_CH4        = Ind_( 'CH4'        )
+    id_CH4_adv    = Ind_( 'CH4',   'A' )
```

Comment:
What is the difference between the adv and non-adv ids? A note in the comments would be helpful to understand.

Reply:
@lizziel: This is used as a test whether CH4 is an advected species or not.  If not, we can exclude reading in fields for CH4, etc. with other species.